prime-dotnet-webapi/Services/Clients/Email/CHESClient.cs

```diff
@@ -0,0 +1,146 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Net.Mail;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+
+namespace Prime.Services
+{
+    public class ChesClient : BaseService, IChesClient
+    {
+        private static HttpClient _client;
+
+        public ChesClient(
+            ApiDbContext context,
+            IHttpContextAccessor httpContext,
+            HttpClient httpClient)
+            : base(context, httpContext)
+        {
+            _client = httpClient ?? throw new ArgumentNullException(nameof(httpClient));
+        }
+
+        public async Task SendAsync(string from, IEnumerable<string> to, IEnumerable<string> cc, string subject, string body, IEnumerable<(string Filename, byte[] Content)> attachments)
+        {
+            var chesAttachments = new List<ChesAttachment>();
+            foreach (var attachment in attachments)
+            {
+                var chesAttachment = new ChesAttachment()
+                {
+                    content = Convert.ToBase64String(attachment.Content),
+                    contentType = "application/pdf",
+                    encoding = "base64",
+                    filename = attachment.Filename
+                };
+
+                chesAttachments.Add(chesAttachment);
+            }
+
+            var requestParams = new CHESEmailRequestParams(from, to, subject, body, chesAttachments);
+
+            var requestContent = new StringContent(JsonConvert.SerializeObject(requestParams), Encoding.UTF8, "application/json");
+
+            HttpResponseMessage response = null;
+            try
+            {
+                response = await _client.PostAsync("email", requestContent);
+
+                if (response.IsSuccessStatusCode)
+                {
+                    var responseJsonString = await response.Content.ReadAsStringAsync();
+                    var successResponse = JsonConvert.DeserializeObject<EmailSuccessResponse>(responseJsonString);
+                }
+            }
+            catch (Exception ex)
+            {
+                throw new Exception("Error occurred when calling CHES Email API. Try again later.", ex);
+            }
+        }
+
+        public async Task<bool> HealthCheckAsync()
+        {
+            HttpResponseMessage response = null;
+            try
+            {
+                response = await _client.GetAsync(new Uri("health"));
```

Comment:
Relative URLs are specified using a string, so this should just be 
`response = await _client.GetAsync("health");`